solutions/cses-1158.mdx

```diff
@@ -14,7 +14,47 @@ author: Sofia Yang
 ## Implementation
 
 <LanguageSection>
+	
+<CPPSection>
 
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+int main() {
+	int n, x;
+	cin >> n >> x;
+	vector<int> cost(n + 1);
+	vector<int> pages(n + 1);
+
+	for (int i = 1; i <= n; i++) {
+		cin >> cost[i];
+	}
+	for (int i = 1; i <= n; i++) {
+		cin >> pages[i];
+	}
+
+	vector<vector<int>> dp(n + 1, vector<int>(x + 1));
```

Comment:
move that multiline comment here

Reply:
you said remove spaces, so i assumed it meant everywhere in places like that. anyways, i changed it

Reply:
bump